import_export/widgets.py

```diff
@@ -368,13 +369,23 @@ class BookResource(resources.ModelResource):
 
             class Meta:
                 fields = ('author',)
+    
+    Additionally, if the model of the foreign key field has natural key and
+    its manager has get_by_natural_key defined per the Django capability,
+    use_natural_foreign_keys can be passed which will cause those functions
+    to be used to identify the related field for export and import respectively
+    See: https://docs.djangoproject.com/en/4.0/topics/serialization/
```

Comment:
I suggest moving this description to rst docs (and adding appropriate example).

Reply:
As far as I can tell the rst for widgets api is imported from the docstrings directly. 

`
.. autoclass:: import_export.widgets.ForeignKeyWidget
   :members:
`

I am not sure exactly where it would fit in the actually written docs, since for almost everything detailed about widgets they refer to the autobuilt api_widgets.rst.

Reply:
I would suggest adding a subsection to `getting_started.rst` - I would say it just needs to be a short summary of the feature and perhaps an example.
thanks

Reply:
That makes sense. I'll add a section on natural keys that references the widgets area and move that chunk out of the foreign key widget. It'll be usefulf for expanding the natural key capability later as well.

Reply:
Pushed that doc update. I should have some time to work on adding the resource option flag to use natural keys at the modelresource level over the next few weeks I think.

Reply:
Great - thanks for this.  I am away for the next few days, so it should get merged when I'm back.  I'm going to ask one of the other maintainers to take a look as well.